downstream/assemblies/platform/assembly-edge-manager-images-.adoc

```diff
@@ -33,3 +33,11 @@ include::platform/con-edge-manager-os-img-script.adoc[leveloffset=+2]
 .Additional resources
 
 * See https://bootc-dev.github.io/bootc/building/guidance.html[Generic guidance for building images] in the bootc documentation.
```

Comment:
Suggest including the link tag: 

link:https://bootc-dev.github.io/bootc/building/guidance.html[Generic guidance for building images] in the bootc documentation.